services/src/main/java/org/keycloak/authentication/AuthenticationProcessor.java

```diff
@@ -854,6 +855,9 @@ public void validateUser(UserModel authenticatedUser) {
         if (authenticatedUser == null) return;
         if (!authenticatedUser.isEnabled()) throw new AuthenticationFlowException(AuthenticationFlowError.USER_DISABLED);
         if (realm.isBruteForceProtected()) {
+            if (event.getEvent().getType().equals(EventType.RESET_PASSWORD)) {
```

Comment:
@stianst hmmm there's no authentication logic required to send a forgot password e-mail, right? So I don't get why this is bad.

What would be a good approach?